### PR TITLE
Fix bug 1397893: Show project and locale name in title element of the…

### PR DESCRIPTION
**Pull request by jotes**
Hi @mathjazz, could you review? Thanks in advance.

### CODE REVIEW

File: pontoon/base/static/js/translate.js [hunk 1]
```diff
@@ -3962,7 +3973,7 @@ var Pontoon = (function (my) {
           queryParams = {};
 
       self.state = state;
-
+      self.updateTitle();
```
Comment (mathjazz) [inline pontoon/base/static/js/translate.js]: You should add a blank line after this statement, to separate the block from the next one, which starts with a comment.

Actually, `pushState` is not the place to put `updateTitle` in - it's called everytime you open a new string for editing or load new entities, and we don't need to update the title on this occasion.

You should move it to the end of the `processEntities()` function.

**Comment by jotes** [comment]
@mathjazz done.

File: pontoon/base/static/js/translate.js [hunk 2]
```diff
@@ -4117,6 +4130,7 @@ Pontoon.attachEntityListHandlers();
 Pontoon.attachEditorHandlers();
 Pontoon.attachBatchEditorHandlers();
 
+Pontoon.updateTitle();
```
Comment (mathjazz) [inline pontoon/base/static/js/translate.js]: We don't need that anymore.

File: pontoon/base/static/js/translate.js [hunk 3]
```diff
@@ -3953,6 +3955,17 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Update title of the current view.
+     */
+    updateTitle: function() {
```
Comment (jotes) [inline pontoon/base/static/js/translate.js]: @mathjazz Do you wish to keep that function or should I move it into  process entities?

Comment (mathjazz) [reply pontoon/base/static/js/translate.js]: I'd keep it. The PR looks OK to me.

**Pull request merged**